src/Database.hs

```diff
@@ -1,9 +1,11 @@
-{-# LANGUAGE DeriveGeneric, DeriveAnyClass #-}
-{-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE QuasiQuotes #-}
 
 module Database where
 
+import Control.Concurrent.STM (readTVarIO)
```

Comment:
Generally you want to import stuff like this from the corresponding package/module (in this case, `stm`) instead of from under the `GHC` namespace (so like `GHC.Conc`).